### ISSUE TITLE
Open location file but not in the vault

### ISSUE THREAD

**Issue by shitake2333** [elias-sundqvist/obsidian-annotator#364]
I hope to specify an external folder as the library for annotator.
If possible, I hope to see it linked with [calibre](https://calibre-ebook.com/)

**Comment by Jopp-gh** [comment]
I agree, it should be possible to add full paths for files outside vaults.

Currently, we're able to load external PDFs with the "open file" button in Annotator, yes. But this change isn't permanently, only temporarily.

Because if we switch to another note and come back to our PDF, Annotator doesn't remember our last loaded pdf. 

Therefore, let's add the option to consider full paths besides vault paths. Maybe add another var for full paths, example: `annotation-external`.
